src/gowrapper/ring/ring.go

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package ring
+
+/*
+#include <stdint.h>
+*/
+import "C"
+
+import (
+	"lattigo-cpp/marshal"
+	"lattigo-cpp/utils"
+	"unsafe"
+
+	"github.com/ldsec/lattigo/v2/ring"
+)
+
+// https://github.com/golang/go/issues/35715#issuecomment-791039692
+type Handle14 = uint64
+
+func getStoredRing(ringHandle Handle14) *ring.Ring {
+	ref := marshal.CrossLangObjMap.Get(ringHandle)
+	return (*ring.Ring)(ref.Ptr)
+}
+
+func GetStoredPoly(polyHandle Handle14) *ring.Poly {
+	ref := marshal.CrossLangObjMap.Get(polyHandle)
+	return (*ring.Poly)(ref.Ptr)
+}
+
+func getStoredUniformSampler(samplerHandle Handle14) *ring.UniformSampler {
+	ref := marshal.CrossLangObjMap.Get(samplerHandle)
+	return (*ring.UniformSampler)(ref.Ptr)
+}
+
+//export lattigo_newRing
+func lattigo_newRing(n int, moduli *C.uint64_t, moduliLen uint64) Handle14 {
```

Comment:
Use `uint64_t` for cross-language consistency

Reply:
Do you mean use `uint64 ` instead of `int`, or do you mean use `C. uint64_t` instead of both `int` and `uint64`?